pkg/framework/test/integration/integration_test.go

```diff
@@ -0,0 +1,53 @@
+package integration_test
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubectl/pkg/framework/test"
+)
+
+var _ = Describe("Integration", func() {
+	It("Successfully manages the fixtures lifecycle", func() {
+		assetsDir, ok := os.LookupEnv("KUBE_ASSETS_DIR")
```

Comment:
Are you planning on having a generic way to give the binaries? `KUBE_TEST_APISERVER` and `KUBE_TEST_ETCD`, so that my test can easily switch the underlying binary?

Reply:
Yes. We have that in the back of our heads. We have [one story](https://www.pivotaltracker.com/story/show/153178125) in our backlog for that.